buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```
@@ -502,7 +504,11 @@ def package_conda_options(platform, base_or_workbench) {
     package_flags = "--build-mantid --build-qt ${docs_flags}"
   }
   else if(base_or_workbench == 'workbench') {
-    package_flags = "--build-workbench"
+    if(WORKBENCH_INCLUDE_DOCS) {
+      package_flags = "--build-workbench"
+    } else {
+      package_flags = "--build-workbench --build-workbench-without-docs"
```

```suggestion
      package_flags = "--build-workbench-without-docs"
```
I think it would be nicer if the argument `--build-workbench-without-docs` alone would specify that you want to build workbench with no docs.
It would remove any ambiguity - someone reading the code might think it's building workbench both with and without docs.